mesheryctl/internal/cli/root/version.go

```diff
@@ -63,7 +63,7 @@ var versionCmd = &cobra.Command{
 		build := mctlCfg.GetVersion().GetBuild()
 		commitsha := mctlCfg.GetVersion().GetCommitSHA()
 
-		logrus.Infof("Client Version: %v \t  GitSHA: %v", build, commitsha)
+		logrus.Infof("Client Version: %v \t  GitSHA: %v", version.GetBuild(), version.GetCommitSHA())
```

Comment:
Hey @UdokaVrede , You can simply use build and commitsha instead of using version.GetBuild() or version.GetCommitSha().